python/pyspark/sql/tests.py

```diff
@@ -3367,6 +3367,24 @@ def test_ignore_column_of_all_nulls(self):
         finally:
             shutil.rmtree(path)
 
+    def test_datasource_with_udf_filter_lit_input(self):
```

Comment:
Add another test case for arrow-based pandas udf?

Reply:
Make sense. Will add.